generators/app/templates/package.json

```diff
@@ -15,12 +15,19 @@
     "postinstall":"ls | grep -v <%=elementName%> | xargs rm -rf"<% } else { %>
     "clean": "rm -rf bower_components/<%=elementName%>",
     "test" : "npm run clean && gulp build && wct",
+    <% if (elementVersion === '2.0') { %>
+    "analyze" : "polymer analyze > analysis.json",
+    "start" : "npm run clean && npm run analyze && gulp default",
```

Comment:
It would be cool if we could also clean up the white space on things like `"analyze" : "polymer` to `"analyze": "polymer` and then get it uniform across the file.